modules/ossm-rn-new-features.adoc

```diff
@@ -15,6 +15,193 @@ Module included in the following assemblies:
+
+[id="new-features-ossm-2-6"]
+== New features {SMProductName} version 2.6
+
+This release of {SMProductName} adds new features, addresses Common Vulnerabilities and Exposures (CVEs), and is supported on {product-title} 4.12 and later.
+
+This release ends maintenance support for {SMProductName} version 2.3. If you are using {SMProductShortName} version 2.3, you should update to a supported version.
+
+include::snippets/ossm-current-version-support-snippet.adoc[]
+//FIPS messaging verified with Matt Werner, CS, OCP on 06/27/2024 via Slack. It is also the same FIPS messaging currently used by Serverless.
+//Per Scott Dodson on 07/15/204 via Slack, confirmed that RHEL 2.9 has been submitted for FIPS validation. Admonition updated accordingly.
+//Per Kirsten Newcomer on 07/16/2024 via Slack, FIPS messaging for Service Mesh has been changed. Jamie (PM) has agreed with change.
+[IMPORTANT]
+====
+{SMProductName} is designed for FIPS. {SMProductShortName} uses the RHEL cryptographic libraries that have been submitted to NIST for FIPS 140-2/140-3 Validation on the x86_64, ppc64le, and s390x architectures.
+For more information about the NIST validation program, see link:https://csrc.nist.gov/Projects/cryptographic-module-validation-program/validated-modules[Cryptographic Module Validation Program]. For the latest NIST status for the individual versions of RHEL cryptographic libraries that have been submitted for validation, see link:https://access.redhat.com/articles/compliance_activities_and_gov_standards#fips-140-2-and-fips-140-3-2[Compliance Activities and Government Standards].
+====
+
+[id="component-versions-ossm-2-6"]
+=== Component versions for {SMProductName} version 2.6
+
+//Component versions updated 07/25/2024.
+//Component tables for 2.5.3 and 2.4.9 updated 07/25/2024
+
+|===
+|Component |Version
+
+|Istio
+|1.20.8
+
+|Envoy Proxy
+|1.28.5
+
+|Kiali
+|1.73.9
+|===
+
+[id="istio-1-20-support-ossm-2-6"]
+=== Istio 1.20 support
+//Jamie
+//OSSM 2.6 supports both Istio 1.19 and Istio 1.20 but only include Istio 1.20.
+//Listing 2 items that are supported instead of only what is not supported.
+
+Service Mesh 2.6 is based on Istio 1.20, which brings in new features and product enhancements, including:
+
+* Native sidecars are supported on {product-title} 4.16 or later.
++
+.Example `ServiceMeshControlPlane` resource
+[source,yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+spec:
+  runtime:
+    components:
+      pilot:
+        container:
+          env:
+            ENABLE_NATIVE_SIDECARS: "true"
+----
+
+* Traffic mirroring in Istio 1.20 now supports multiple destinations. This feature enables the mirroring of traffic to various endpoints, allowing for simultaneous observation across different service versions or configurations.
+
+While {SMProductName} supports many Istio 1.20 features, the following exceptions should be noted:
```

Comment:
Hey @lpettyjo ,

Out of curiosity, since previous entries for a 2.x releases (2.5, for example, says "Service Mesh 2.5 is based on Istio 1.18, which brings in new features and product enhancements. While Red Hat OpenShift Service Mesh supports many Istio 1.18 features, the following exceptions should be noted:") does that now need to be updated to be consistent? 

Changing it as you suggest introduces an inconsistency, so am curious.

Reply:
This change introduces an inconsistency in OSSM docs, and is outside the scope of this PR. A Jira has been created to investigate this change and added to the OSSM 2.x Doc Maintenance Jira Epic.